client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -31,23 +38,36 @@ const CheckoutThankYouHeader = React.createClass( {
 
 	getText() {
 		if ( ! this.props.isDataLoaded || ! this.props.primaryPurchase ) {
-			return this.translate( "You will receive an email confirmation shortly. What's next?" );
+			return this.translate( 'You will receive an email confirmation shortly.' );
 		}
 
-		if ( isFreeTrial( this.props.primaryPurchase ) ) {
-			return this.translate( "We hope you enjoy {{strong}}%(productName)s{{/strong}}. What's next? Take it for a spin!", {
-				args: {
-					productName: this.props.primaryPurchase.productName
-				},
-				components: {
-					strong: <strong />
-				}
-			} );
-		} else if ( isPlan( this.props.primaryPurchase ) ) {
+		if ( isPlan( this.props.primaryPurchase ) ) {
 			return this.translate( "Your site is now on the {{strong}}%(productName)s{{/strong}} plan. It's doing somersaults in excitement!", {
 				args: { productName: this.props.primaryPurchase.productName },
 				components: { strong: <strong /> }
 			} );
+		} else if ( isDomainRegistration( this.props.primaryPurchase ) ) {
```

Comment:
I think we should just use `if` statements here – we have a linting rule for not using an else after the previous clause ends in a return.